vg/build/Dockerfile

```diff
@@ -20,6 +24,7 @@ WORKDIR /home
 RUN git clone --recursive https://github.com/vgteam/vg.git
 
 WORKDIR /home/vg
-RUN git checkout 9aa20df8f71ebf67b0050e3c77db5f6e4cab256d
```

Comment:
Yes, you need to pin the commit you are checking out. Especially, if you are tagging the image as a specific version, i.e. v1.4.0-1649-g8a28f7d.